src/ecdsa/eddsa.py

```diff
@@ -116,6 +116,12 @@ def __ne__(self, other):
     def point(self):
         return self.__point
 
+    @point.setter
+    def point(self, other):
+        if self.__point != other:
```

Comment:
Point doesn't have assigned generator, so it can't change, changing order of the point is explicitly allowed, I'm pretty sure that `INFINITY` will not be allowed...